docs/docs/glamor.md

```diff
@@ -2,4 +2,4 @@
 title: Glamor
 ---
 
-CSS-in-JS library [Glamor](https://github.com/threepointone/glamor) is not actively maintained. the maintainer recommends using [Emotion](/docs/emotion/).
+CSS-in-JS लाइब्रेरी में [Glamor](https://github.com/threepointone/glamor) सक्रिय रूप से बनाई नहीं रखी गई है। मेंटेनर [Emotion](/docs/emotion/) का उपयोग करने की सलाह देता है।
```

Comment:
`बनाई नहीं रखी गई` => `मेन्टेन नहीं की जाती`

Reply:
Done with the changes.